app/controllers/admin_controller.rb

```diff
@@ -75,4 +92,21 @@ def email_alert_params
       :signup_link,
     )
   end
+
+  def facets_params
+    allowed_facets = %i[
```

Comment:
Suggesting slightly more specificity - `allowed_facets` to me suggests we're specifying what facets are allowed rather than what attributes a facet can have:
```suggestion
    allowed_facet_params = %i[
```